sentry-spring/src/main/java/io/sentry/spring/reactive/SentryReactiveWebHelper.java

```diff
@@ -0,0 +1,23 @@
+package io.sentry.spring.reactive;
+
+import io.sentry.IHub;
+import io.sentry.ILogger;
+import io.sentry.SentryLevel;
+import io.sentry.SystemOutLogger;
+import java.util.function.Consumer;
+import org.springframework.web.server.ServerWebExchange;
+
+public final class SentryReactiveWebHelper {
+  static final String REQUEST_HUB_ATTR_NAME = "SentryReactiveWebHelper.REQUEST_HUB_ATTR_NAME";
+
+  private static final ILogger LOGGER = new SystemOutLogger();
+
+  public static void withRequestHub(ServerWebExchange exchange, Consumer<IHub> hubConsumer) {
+    Object ihub = exchange.getAttributes().get(REQUEST_HUB_ATTR_NAME);
+    if (ihub instanceof IHub) {
+      hubConsumer.accept((IHub) ihub);
+    } else {
+      LOGGER.log(SentryLevel.WARNING, "No Hub configured in ServerWebExchange");
```

Comment:
Should this be an error instead?
This integration wouldn't function in this state, right?
```suggestion
      LOGGER.log(SentryLevel.ERROR, "No Hub configured in ServerWebExchange");
```